src/renderer/components/Dropdown/Dropdown.tsx

```diff
@@ -31,19 +36,31 @@ export const Dropdown: React.FunctionComponent<IDropdownProps> = (props: IDropdo
                     </div>
                     <div className="dropdown-items-container">
                         <div className="dropdown-items">
-                            {props.options.map(option =>{
-                                return <div 
-                                    key={option} 
-                                    onClick={(): void => {props.onChange(getSelectedIndex(option));showHide();}} 
-                                    className={`dropdown-item 
-                                ${visible}
-                                ${getSelectedIndex(option)===props.current?"selected":""}`} 
-                                >{option}</div>;
-                            })}
+                            { Array.isArray(props.options) ?
```

Comment:
Wouldn't it be easier if you convert array to object and call onChange with key?
Than you don't need getSelectedIndex as key will always be either prop in object or index in array?